tests/sentry/issues/endpoints/test_organization_group_index.py

```diff
@@ -3917,6 +3920,58 @@ def test_snuba_heavy_error_handled_boolean(self, _: MagicMock) -> None:
         assert len(response_handled_0.data) == 1
         assert int(response_handled_0.data[0]["id"]) == handled_event.group.id
 
+    def test_feedback_filtered_by_default(self, _):
+        with Feature(
+            {
+                "organizations:feedback-ingest": True,
+                "organizations:feedback-visible": True,
+            }
+        ):
+            event = self.store_event(
+                data={"event_id": uuid4().hex, "timestamp": iso_format(before_now(seconds=1))},
+                project_id=self.project.id,
+            )
+            assert event.group is not None
+
+            feedback_event = mock_feedback_event(self.project.id, before_now(seconds=1))
+            create_feedback_issue(
+                feedback_event, self.project.id, FeedbackCreationSource.NEW_FEEDBACK_ENVELOPE
+            )
+            self.login_as(user=self.user)
+            res = self.get_success_response()
+
+        assert len(res.data) == 1
+        issue = res.data[0]
+        feedback_group = Group.objects.get(type=FeedbackGroup.type_id)
+        assert int(issue["id"]) != feedback_group.id
```

Comment:
could we add a comment here explaining what's being tested? (for clarity)
something simple like 
`# test that the issue returned is not the feedback issue`